ext/dom/private_data.c

```diff
@@ -137,4 +142,28 @@ void php_dom_remove_templated_content(php_dom_private_data *private_data, const
 	}
 }
 
+zend_long php_dom_get_template_count(const php_dom_private_data *private_data)
```

Comment:
Should this return a `uint32_t` instead?

Reply:
Hmm probably